include/struct_defs/pokemon_data_block_shuffle_0.h

```diff
@@ -0,0 +1,16 @@
+#ifndef POKEPLATINUM_STRUCT_0207825C_H
+#define POKEPLATINUM_STRUCT_0207825C_H
+
+#include "struct_defs/pokemon_data_block_a.h"
+#include "struct_defs/pokemon_data_block_b.h"
+#include "struct_defs/pokemon_data_block_c.h"
+#include "struct_defs/pokemon_data_block_d.h"
+
+typedef struct PokemonDataBlocksShuffle0 {
+    PokemonDataBlockA dataBlockA;
+    PokemonDataBlockB dataBlockB;
+    PokemonDataBlockC dataBlockC;
+    PokemonDataBlockD dataBlockD;
+} PokemonDataBlocksShuffle0;
```

Comment:
This is a very odd construction. Is there a way that we can dynamically specify these?